### DISCUSSION TITLE
How to output accumulated precipitation?

### DISCUSSION THREAD

**Discussion by YongjunZHENG** [ufs-community/ufs-weather-model#2016] [Q&A] [answered]
## Description
How do I output accumulated precipitation from the UFS WM?

**Comment by BrianCurtis-NOAA** [comment]
@YongjunZHENG I've converted your issue into a discussion as it seems this could benefit from such a style. We can create an issue from this if needed later on.

**Comment by jkbk2004** [comment]
@JessicaMeixner-NOAA @lisa-bengtsson @grantfirl is it possible to output accumulated precipitation through diag_table?

**Reply by JessicaMeixner-NOAA** [reply]
I don't think I'm the correct person to help with this question.  Hopefully others will chime in.

**Comment by lisa-bengtsson** [comment, marked as answer]
yes, this is the accumulated precipitation from start to end of forecast in "diag_table":
"gfs_phys",    "totprcp_ave",   "prate_ave",     "fv3_history2d",  "all",  .false.,  "none",  2

It is different from e.g. prateb_ave which is the "bucket" precipitation averaged over the bucket determined by fhzero: 
"gfs_phys",    "totprcpb_ave",  "prateb_ave",    "fv3_history2d",  "all",  .false.,  "none",  2

**Comment by gspetro-NOAA** [comment]
@YongjunZHENG Did Lisa's answer resolve your question? If so, you can click "Mark as answer." If not, let us know so that we can get you more information! :)
